### PR TITLE
Remove onboarding experiment

### PR DESCRIPTION
**Pull request by bwaresiak**
Task/Issue URL: https://app.asana.com/0/856498667320406/1162322702538942
Tech Design URL:
CC:

**Description**:

- Removed experiment setup.
- Removed CTA alerts.

Note: CTA Reminder toast is still there - we show it in case user did not watch the onboarding video till the end.

**Steps to test this PR**:
1. Run the onboarding.
2. Home Row CTA should be the last step of the onboarding flow.
3. No alerts should be shown.
4. Floating tips should work.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: DuckDuckGo/HomeViewController.swift [hunk 1]
```diff
@@ -121,22 +120,12 @@ class HomeViewController: UIViewController {
...
-            showHomeRowCTA()
-        }
-        
         installHomeScreenTips()
-        
         viewHasAppeared = true
     }
     
-    func resetHomeRowCTAAnimations(variantManager: VariantManager = DefaultVariantManager()) {
+    func resetHomeRowCTAAnimations() {
```
Comment (subsymbolic) [inline DuckDuckGo/HomeViewController.swift]: Does `resetHomeRowCTAAnimations` still make sense as a method name?

File: DuckDuckGo/UnifiedAddToHomeRowCTAViewController.swift [hunk 2]
```diff
@@ -1,223 +0,0 @@
...
-    }
-    
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
-        animateOnFirstAppearance()
-        startVideo()
-    }
-
-    @IBAction func gotItButtonPressed() {
-        Pixel.fire(pixel: .homeRowCTAGotItTapped)
```
Comment (subsymbolic) [inline DuckDuckGo/UnifiedAddToHomeRowCTAViewController.swift:69]: Delete unused `homeRowCTA` pixels.

**Review by subsymbolic — APPROVED** [review]
A couple of small suggestions but no need for a rereview when these are done, merge when you're ready.

I also noticed a small landscape glitch in the reminder screen which should be fixed however as it's an edge-case I'm happy to include it in a separate PR to let this one go out in a release.

reminder_landscape_glitch

**Comment by bwaresiak** [comment]
@subsymbolic Thank you! I've committed tweaks and created https://app.asana.com/0/856498667320406/1162336160858417 as a follow up for the landscape glitch.

**Pull request merged**